internal/ui/etui/ephemeral_tui.go

```diff
@@ -73,16 +75,25 @@ func OutputToEphemeralTUI(workerErrs <-chan error, subscription *partybus.Subscr
 	ctx := context.Background()
 	grypeUIHandler := grypeUI.NewHandler()
 
-eventLoop:
+	var errResult error
 	for {
 		select {
-		case err := <-workerErrs:
+		case err, ok := <-workerErrs:
 			if err != nil {
+				if errors.Is(err, grypeerr.ErrAboveSeverityThreshold) {
+					errResult = err
```

Comment:
Definitely up for discussion --> if I'm reading this correctly, it looks like the logic around `errResult` is such that errors that aren't the latest error are lost forever. Is that true? If so, would it be worth collecting surfaced errors for reporting, if we're not displaying them immediately?

Reply:
this is here to prevent return of an error and short circuit the UI logic --per our conversation, this section needs a second pass on design. It is OK that we are capturing the most recent `errResult` since this is only there to indicate `ErrAboveSeverityThreshold` --will not scale though.